Makefile

```diff
@@ -3,6 +3,7 @@ SHELL := bash
 .SHELLFLAGS := -eu -o pipefail -c
 .DELETE_ON_ERROR:
 .SUFFIXES:
+.ONESHELL:
```

Comment:
This configures make to execute each recipe as a single shell script rather than executing each line in a separate shell: 
 * https://www.gnu.org/software/make/manual/html_node/Special-Targets.html